### PR TITLE
[Feature] redesign plugin settings to look like other Jellyfin settings.

### PR DESCRIPTION
**Pull request by paranoyed**
Hi, 

first of all thank you for developing a LDAP Plugin for Jellyfin which is OpenSource!

This PR redesigns the plugin settings to look like other Jellyfin settings.

### CODE REVIEW

**Comment by joshuaboniface** [comment]
Can you post a screenshot of the before/after design changes for posterity?

**Comment by paranoyed** [comment]
Shure!
Before: https://imgur.com/cZJzn4B
After: https://imgur.com/X3r33RD

**Comment by hawken93** [comment]
Lgtm if it's still functional. Looks nice

File: LDAP-Auth/Config/configPage.html [hunk 1]
```diff
@@ -8,108 +8,74 @@
...
-                            <label for="txtLdapServer">
-                                LDAP Server:
-                            </label>
-                            <input type="text" id="txtLdapServer" name="txtLdapServer" />
-                            <div class="fieldDescription">
-                                The Servername for the LDAP server you wish to use for Authentications
+                    <div class="verticalSection verticalSection-extrabottompadding">
+                        <div class="sectionTitleContainer flex align-items-center">
+                            <h2 class="sectionTitle">LDAP Settings:</h2>
+                            <a is="emby-linkbutton" class="raised button-alt headerHelpButton" target="_blank" href="https://github.com/jellyfin/jellyfin-plugin-ldapauth">${Help}</a>
```
Comment (cvium) [inline LDAP-Auth/Config/configPage.html]: `emby-linkbutton` is going away. Use `emby-button` instead

Comment (paranoyed) [reply LDAP-Auth/Config/configPage.html]: ok, fixed it.

**Pull request merged**